components/Polls/PollList.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { ListView, View, TextInput, TouchableOpacity, NavigatorIOS } from 'react-native';
+import { Container, Content, Text, Button, ListItem, Icon } from 'native-base';
+import { firebaseDB } from '../../firebase/firebaseHelpers';
+import Poll from './Poll';
+
+export default class PollList extends Component {
+  //input new poll name
+  //submit button to create a new poll entry in db
+    //anncounce it in chat
+    //forward to Poll component
+
+  //show list of active polls
+  //allow poll deletion
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: this.props.user ? this.props.user.displayName : 'Anonymous',
```

Comment:
Is there ever a case where we wouldn't pass in a user and/or group? Why the check?